repo/packages/R/riak/1/command.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@ryadav88 I have some questions about removing `config.json` and migration to `cli` inside `resource.json`.
For package version `2.0` we had this https://github.com/basho-labs/riak-mesos-dcos-shims/blob/0.0.1/setup.py for the DCOS cli. Unfortunately, i did not find any examples or tutorials how to migrate from old solution(2.0) to the new one(3.0).

May you help with this? 

Thank you in advance